### ISSUE TITLE
toolbar icons still blue even though #disable-thorium-icons flag is enabled

### ISSUE THREAD

**Issue by mohab001** [Alex313031/Thorium-Win#296]
**System Details**
 - OS: [Windows 10]
 - Thorium Version [128.0.6613.189]

**Problem**
Home button icon and extensions icon are still blue even though #disable-thorium-icons flag is enabled (it only changes reload icon)

**Screenshots**
![image](https://github.com/user-attachments/assets/02c4ad92-2b55-4bbc-add5-34ccec649d8c)

**Comment by gz83** [comment]
This is intentional

**Comment by mohab001** [comment]
It would be nice to have the option to make toolbars icons color decided by themes, or at least add the option to revert them back to the default gray color, cause it looks really ugly with some themes.
Also the flag is kinda pointless if it changes only one icon.

**Comment by Alex313031** [comment]
@gz83 @mohab001 Nope, this IS a bug. The home button is using the incorrect .icon file. I will fix.

**Comment by Alex313031** [comment]
@gz83 In fact, next release I have to rebase ALL the .icon files, as they changed the default line width and font size > https://source.chromium.org/chromium/chromium/src/+/f824ddc5635f14e9fe879df062d8fbd66251f312

**Comment by mezhgano** [comment]
I'm on latest release (M128.0.6613.189) with **#disable-thorium-icons** flag enabled. 
Player and tab mute icons is violet:

![Screenshot 2024-11-09 14-20-50_-_YouTube_-_Thor](https://github.com/user-attachments/assets/e968dd22-6d23-4746-807c-7b98424d6b5a)

**Comment by gz83** [comment]
Please try the latest version 138. If you still have problems, please let us know.
